test/jdk/javax/swing/JButton/HtmlButtonImageTest/HtmlButtonImageTest.java

```diff
@@ -110,6 +116,15 @@ private static void testImageCentering(int... colors) throws IOException {
                 throw new RuntimeException("HTML image not centered in button");
```

Comment:
overall looks ok. Only thing is during failure, it does not show for which L&F the test is failing since you are iterating for all L&Fs.

Reply:
Sure, this can be changed. I can instead iterate through all L&F's and track all failing L&F's in a buffer to be displayed in the runtime exception in the end. How should the images be handled? Should I just generate X images where X is the number of failed L&F's?